### DISCUSSION TITLE
How to add new vector name to existed collection?

### DISCUSSION THREAD

**Discussion by failable** [qdrant/qdrant#2752] [Q&A] [answered]
Can I add new vector name to existed collection? I read the document but could not find a way to add new vector names without delete and recreate the collection.

Thanks.

**Comment by admin** [comment, marked as answer]
**Update:** this is now supported in Qdrant 1.18.0 and up.

https://qdrant.tech/documentation/manage-data/collections/#update-vector-schema

---

that is not currently supported. You would need to create a new collection and import data there

**Comment by wanfade** [comment]
Is there any update in 2025? Does it support add new vector in a existed collection?

**Reply by timvisee** [reply]
This isn't supported at the moment.

**Reply by ellenhp** [reply]
2026 update: https://github.com/qdrant/qdrant-client/issues/1153#issuecomment-4486640486
